src/query/catalog/src/catalog/mod.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// catalog_interface is the module defining `Catalog` trait
+mod interface;
+/// the catalog manager implementation
+mod manager;
+
+pub use interface::*;
+pub use manager::*;
```

Comment:
I prefer to export specified structs like `pub use manager::CatalogManager` so we can avoid exporting unintended types.